mapviz/src/mapviz.cpp

```diff
@@ -1474,6 +1491,7 @@ void Mapviz::RemoveDisplay(QListWidgetItem* item)
   {
     canvas_->RemovePlugin(plugins_[item]);
     plugins_[item] = MapvizPluginPtr();
+    plugins_.erase(item);
```

Comment:
Does adding an empty pointer do anything if it's being erased immediately afterward?